app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+
+  devise :invitable, :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  ROLES = [:admin, :buyer]       
+  enum role: ROLES
+  scope :buyer_role, -> {where(role: 'buyer')}
+
+end
```

Comment:
class User < ApplicationRecord
  devise :invitable, :database_authenticatable, :registerable,
         :recoverable, :rememberable, :validatable

  ROLES = [:admin, :buyer]       
  enum role: ROLES
end

Reply:
remove new lines at 2 and 8 line